### ISSUE TITLE
Addition of swahili Braille in Braile Category of NVDA settings Menu

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#8686]
swahili grades 1 and 2 Braille should be added in Braille menu item. this is so to enable swahili people using Braille displays and embossers with NVDA to well access braille. currently, when you access swahili braille in english braille, it sounds English.

#### Other information about your system:
             Machine name: DESKTOP-R0NVNF4
               Machine Id: {F1EFE585-AA4A-485E-9FE9-4DC0FCCDACBF}
         Operating System: Windows 10 Pro 64-bit (10.0, Build 17134) (17134.rs4_release.180410-1804)
                 Language: English (Regional Setting: English)
      System Manufacturer: TOSHIBA
             System Model: Satellite C850-C035
                     BIOS: InsydeH2O Version 03.72.306.50 (type: BIOS)
                Processor: Intel(R) Core(TM) i3-2348M CPU @ 2.30GHz (4 CPUs), ~2.3GHz
                   Memory: 8192MB RAM
      Available OS Memory: 8074MB RAM
                Page File: 2837MB used, 7156MB available
              Windows Dir: C:\WINDOWS
          DirectX Version: DirectX 12
      
#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
yes, the issue is the same.

**Comment by ehollig** [comment]
This issue may need to be raised on [Liblouis Github](https://github.com/liblouis/liblouis) page if it is not added already. CC @leonardder

**Comment by dkager** [comment]
Indeed, there is no Swahili braille table in liblouis yet. This needs to be added first, then it can be added to NVDA's list.

**Comment by Adriani90** [comment]
Given the comments above, I think it is best to raise a Liblouis issue for this first and then link the issue here so that we can track it. I am closing this issue for now.
